components/dropdown/index.jsx

```diff
@@ -0,0 +1,32 @@
+import { createElement } from "complate-stream";
+
+export default function Dropdown({ id, class: additionalClass, label, contextual = "secondary",
+									 active, disabled, type: typeName = "button", size }, ...children) {
+
+	let classNames = ["dropdown-toggle", "btn", `btn-${contextual}`];
+
+	if (additionalClass) {
+		classNames.push(additionalClass);
+	}
+
+	if (active) {
+		classNames.push("active");
+	}
+
+	if (disabled) {
+		classNames.push("disabled");
+	}
+
+	if (size) {
+		classNames.push(`btn-${size}`);
+	}
```

Comment:
FWIW, this could be much more concise with [classnames](https://github.com/JedWatson/classnames):

```
$ npm install classnames
```

```javascript
import classNames from "classnames";
…
let classes = classNames("dropdown-toggle", "btn", `btn-${contextual}`, {
    [`btn-${size}`]: size,
    active,
    disabled,
    additionalClass
});
…
<button class={classes} …>
```

Reply:
That’s a good point, would you mind opening a new issue for that?  I believe we could use more concision in _all_ of our current components!

Reply:
→ #13